src/artemis/fast_grid_scan_plan.py

```diff
@@ -58,6 +63,19 @@ def do_fgs():
     )
     run_end(datacollection_id)
 
+    xray_centre = wait_for_result(datacollection_id)
+    xray_centre_motor_position = (
+        parameters.grid_scan_params.grid_position_to_motor_position(xray_centre)
+    )
+    yield from bps.mv(
+        sample_motors.x,
+        xray_centre_motor_position.x,
+        sample_motors.y,
+        xray_centre_motor_position.y,
+        sample_motors.z,
+        xray_centre_motor_position.z,
+    )
```

Comment:
Must - Since #141 just got merged the test for this method will also need to be updated to mock this section